services/provider/server/consumer.go

```diff
@@ -54,7 +55,7 @@ func newConsumerManager(ctx context.Context, cl client.Client, namespace string)
 }
 
 // Create creates a new storageConsumer resource, updates the consumer cache and returns the storageConsumer UID
-func (c *ocsConsumerManager) Create(ctx context.Context, onboard ifaces.StorageClientOnboarding) (string, error) {
+func (c *ocsConsumerManager) Create(ctx context.Context, onboard ifaces.StorageClientOnboarding, onboardingTicket *services.OnboardingTicket) (string, error) {
```

Comment:
I am not sure it will benefit us to have the Consumer Manager be aware of the internal structure of the ticket. Up until this change, the ticket string was treated as an opaque string. 

```suggestion
func (c *ocsConsumerManager) Create(ctx context.Context, onboard ifaces.StorageClientOnboarding, int StorageQuotaInGiB) (string, error) {
```